mintpy/save_kite.py

```diff
@@ -158,9 +158,12 @@ def main(iargs=None):
 
     elif attr['FILE_TYPE'] == 'ifgramStack':
         date1, date2 = inps.dset.split('-')[1].split('_')
-
+        
+    elif attr['FILE_TYPE'] == '.unw':
+        date1, date2 = ptime.yyyymmdd(attr['DATE12'].split('-'))
+        
```

Comment:
It seems to me that we don't need a separate `elif` check for `.unw` as it's covered in the `else` block below.

Reply:
@yunjunz you are right, these lines are redundant...I will delete them